### DISCUSSION TITLE
Need some help with Challenge 10

### DISCUSSION THREAD

**Discussion by sahildhar** [github/securitylab#168] [General] [answered]
Hi,

I am new to codeQL and currently going through the following learning challenges. https://lab.github.com/githubtraining/codeql-u-boot-challenge-(cc++). In challenge 10, I am getting 4 false positives even after running the query from the solution. 

More details can be found here:

https://github.com/sahildhar/codeql-uboot/issues/20

**Comment by jbj** [comment]
It doesn't seem right that our prescribed solution is rejected. Ping @kevinbackhouse.

**Comment by sahildhar** [comment]
@kevinbackhouse can you help here ?

**Comment by xcorail** [comment, marked as answer]
There was a mismatch between the CodeQL version used and the proposed solution. The course has been updated and the proposed solution is now passing.

**Reply by sahildhar** [reply]
Thanks @xcorail
